client/client.go

```diff
@@ -22,14 +22,14 @@ type Config struct {
 
 func New() Client {
 	return Client{
-		ldap: &ldaputil.Client{
+		LDAP: &ldaputil.Client{
 			LDAP: ldaputil.NewLDAP(),
 		},
```

Comment:
`Logger` could be set here, and we could move the nil check within this constructor?

Reply:
Done.